app/models/policy_version.rb

```diff
@@ -86,6 +86,61 @@ def before_save
     self.policy_sha256 = Digest::SHA256.hexdigest(policy_text)
   end
 
+  def after_save
+    ensure_versions_limit
+  rescue => e
+    Rails.logger.error("Error while ensuring policy '#{id}' versions retention limit: #{e.inspect}")
```

Comment:
@eranha  is this a log? If so please put Shuli as reviewer
Also for errors i think you need to have an error code and so on.
what is the reason for the error? what do you expect the user to do?

Reply:
This is not expected to happen, but I don't want an error to fail the policy load.